v2/api/network/customizations/private_endpoints_extensions.go

```diff
@@ -60,3 +67,99 @@ func (extension *PrivateEndpointExtension) PostReconcileCheck(
 
 	return extensions.PostReconcileCheckResultSuccess(), nil
 }
+
+func (extension *PrivateEndpointExtension) ExportKubernetesResources(
+	ctx context.Context,
+	obj genruntime.MetaObject,
+	armClient *genericarmclient.GenericClient,
+	log logr.Logger,
+) ([]client.Object, error) {
+	// This has to be the current hub storage version. It will need to be updated
+	// if the hub storage version changes.
+	endpoint, ok := obj.(*network.PrivateEndpoint)
+	if !ok {
+		return nil, errors.Errorf("cannot run on unknown resource type %T, expected *network.PrivateEndpoint", obj)
+	}
+
+	// Type assert that we are the hub type. This will fail to compile if
+	// the hub type has been changed but this extension has not
+	var _ conversion.Hub = endpoint
+
+	hasIpConfiguration := configMapSpecified(endpoint)
```

Comment:
very minor: 
```suggestion
	hasIPConfiguration := configMapSpecified(endpoint)
```

Reply:
Actually maybe just consider 
```suggestion
	hasConfigMaps := configMapSpecified(endpoint)
```

or else rename the `configMapSpecified` to `ipAddressSpecified`?